Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -230,6 +292,7 @@ public boolean isVenmoAppSwitchAvailable(@NonNull Context context) {
      * @param context  Android Context
      * @param callback {@link VenmoIsReadyToPayCallback}
      */
+    // TODO: How should this method behave with web fallback enabled?
```

Comment:
I wonder if this method is still relevant. I'm not sure what the original intent was historically, but it seems like it's a good check to make before redirecting a user to Venmo's page in Google Play?

Reply:
🤔 I would think we can remove this now and just do the config check for `isVenmoEnabled` internally. With the fallback to web flow, we no longer care (nor should merchants) if `isVenmoAppSwitchAvailable` aside from if we want to send analytics in the future if the app is not installed. I would think we can remove this method entirely, but curious to hear what y'all think.

Reply:
Yeah I think this can be removed, since it doesn't really provide any value anymore (shopper insights will likely replace this anyways)